index.html

```diff
@@ -1803,7 +1803,7 @@ <h4>Syntactic search: XPath</h4>
                         The request must contain a valid XPath 3.1 [[xpath-31]] as search parameter.
                         A successful response must have 200 (OK) status, contain
                         <code>application/json</code> Content-Type header, and in
-                        the body a set of complete TDs or a set of TD fragments.
+                        the body, the query response in JSON serialization.
```

Comment:
Perhaps add a sentence: "The data schema for the response is defined implicitly by the query and the XPath specification."

Reply:
```suggestion
                        the body, the query response in JSON serialization.  The data schema for the response is defined implicitly by the query and the XPath specification.
```